mcp/index.js

```
@@ -0,0 +1,243 @@
+        switch (command) {
+            case 'start':
+                await cli.startServer();
+                break;
+
+            case 'status':
+                await cli.getSystemStatus();
+                break;
+
+            case 'list-tasks':
+                const filter = args[1] ? JSON.parse(args[1]) : {};
+                await cli.listTasks(filter);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)